nexus/reconfigurator/planning/tests/output/planner_basic_add_sled_2_3.txt

```diff
@@ -1,48 +1,61 @@
-          10d98a73-ec88-4aff-a7e8-7db6a87880e6 in service     crucible [underlay IP fd00:1122:3344:101::24] (unchanged)
-          2a455c35-eb3c-4c73-ab6c-d0a706e25316 in service     crucible [underlay IP fd00:1122:3344:101::29] (unchanged)
-          3eda924f-22a9-4f3e-9a1b-91d1c47601ab in service     crucible [underlay IP fd00:1122:3344:101::23] (unchanged)
-          587be699-a320-4c79-b320-128d9ecddc0b in service     crucible [underlay IP fd00:1122:3344:101::2b] (unchanged)
-          6fa06115-4959-4913-8e7b-dd70d7651f07 in service     crucible [underlay IP fd00:1122:3344:101::2c] (unchanged)
-          8f3a1cc5-9195-4a30-ad02-b804278fe639 in service     crucible [underlay IP fd00:1122:3344:101::28] (unchanged)
-          a1696cd4-588c-484a-b95b-66e824c0ce05 in service     crucible [underlay IP fd00:1122:3344:101::25] (unchanged)
-          a2079cbc-a69e-41a1-b1e0-fbcb972d03f6 in service     crucible [underlay IP fd00:1122:3344:101::2a] (unchanged)
-          c66ab6d5-ff7a-46d1-9fd0-70cefa352d25 in service     nexus [underlay IP fd00:1122:3344:101::22] (unchanged)
-  sled 590e3034-d946-4166-b0e5-2d0034197a07
-      zone config generation 2
-          18f8fe40-646e-4962-b17a-20e201f3a6e5 in service     crucible [underlay IP fd00:1122:3344:102::2a] (unchanged)
-          47199d48-534c-4267-a654-d2d90e64b498 in service     internal_ntp [underlay IP fd00:1122:3344:102::21] (unchanged)
-          56d5d7cf-db2c-40a3-a775-003241ad4820 in service     crucible [underlay IP fd00:1122:3344:102::29] (unchanged)
-          6af7f4d6-33b6-4eb3-a146-d8e9e4ae9d66 in service     crucible [underlay IP fd00:1122:3344:102::2b] (unchanged)
-          704e1fed-f8d6-4cfa-a470-bad27fdc06d1 in service     nexus [underlay IP fd00:1122:3344:102::22] (unchanged)
-          7a9f60d3-2b66-4547-9b63-7d4f7a8b6382 in service     crucible [underlay IP fd00:1122:3344:102::26] (unchanged)
-          93f2f40c-5616-4d8d-8519-ec6debdcede0 in service     crucible [underlay IP fd00:1122:3344:102::2c] (unchanged)
-          ab7ba6df-d401-40bd-940e-faf57c57aa2a in service     crucible [underlay IP fd00:1122:3344:102::28] (unchanged)
-          af322036-371f-437c-8c08-7f40f3f1403b in service     crucible [underlay IP fd00:1122:3344:102::23] (unchanged)
-          d637264f-6f40-44c2-8b7e-a179430210d2 in service     crucible [underlay IP fd00:1122:3344:102::25] (unchanged)
-          dce226c9-7373-4bfa-8a94-79dc472857a6 in service     crucible [underlay IP fd00:1122:3344:102::27] (unchanged)
-          edabedf3-839c-488d-ad6f-508ffa864674 in service     crucible [underlay IP fd00:1122:3344:102::24] (unchanged)
-+ sled b59ec570-2abb-4017-80ce-129d94e7a025 (added)
-+     zone config generation 2
-+         2d73d30e-ca47-46a8-9c12-917d4ab824b6 in service     internal_ntp [underlay IP fd00:1122:3344:104::21] (added)
+from: blueprint 979ef428-0bdd-4622-8a72-0719e942b415
+-       external DNS version: 1
+-       internal DNS version: 1
```

Comment:
It seems like this shouldn't go in between the "from"/"to" labels.  (Or, why would DNS be different than the zones in this way?)

Reply:
Hmmm -- so are you saying this should also be a table?

Reply:
I don't know?  It's more about the order of things.  Right now it's:

```
from: blueprint 1 label
     some stuff from blueprint1 that may have changed (the DNS stuff)
to: blueprint 2 label
     some stuff from blueprint2 that may have changed (the DNS stuff)

more stuff from blueprint1 that may have changed (the non-DNS stuff)
more stuff from blueprint2 that may have changed (the non-DNS stuff)
```

It makes it look like the DNS stuff is part of the identifying information about the blueprint, but I think of it as just another part of the blueprint contents.  I'd expect it to look more like:

```
from: blueprint1 label
to: blueprint2 label

DNS changes from blueprint1 to blueprint2
Omicron zone changes from blueprint1 to blueprint2
... (other changes from blueprint1 to blueprint2)
```

or more concretely instead of:

```
from: blueprint 979ef428-0bdd-4622-8a72-0719e942b415
-       external DNS version: 1
-       internal DNS version: 1
to:   blueprint 4171ad05-89dd-474b-846b-b007e4346366
+       external DNS version: 1
+       internal DNS version: 1

zones:
```

it'd be:

```
from: blueprint 979ef428-0bdd-4622-8a72-0719e942b415
to:   blueprint 4171ad05-89dd-474b-846b-b007e4346366

DNS:
    - internal DNS version: 1
    + internal DNS version: 2
    - external DNS version: 1
    + external DNS version: 2

zones:
```

(or something more concise if we want, like `internal DNS version: 1 -> 2`, but the point is that "from"/"to" would be the top two lines)

Reply:
Ahh, I see what you mean -- misunderstood earlier. That definitely makes sense, and I'll include it with @jgallagher's suggestion to move this metadata to the bottom.

Reply:
Done, and moved to the end.